src/main/java/com/j256/simplemagic/entries/MagicType.java

```diff
@@ -130,7 +130,7 @@
 		}
 	}
 
-	private MagicType(String name, MagicMatcher matcher) {
+	MagicType(String name, MagicMatcher matcher) {
```

Comment:
Why remove private here?  Is it a bug?  Why make this change?

Reply:
as said in the commit message: enums constructors are already provate